### PR TITLE
Add omines/antispam-bundle

### PR DESCRIPTION
**Pull request by curry684**
| Q             | A
| ------------- | ---
| License       | MIT
| Packagist     | https://packagist.org/packages/omines/antispam-bundle

### CODE REVIEW

**Comment by curry684** [comment]
QA check on PHP 7.4 is failing because the bundle requires 8.1 or later.

File: omines/antispam-bundle/0.1/config/packages/antispam.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+            # Reject text fields that consist for more than 40% of Cyrillic (Russian) characters
+#            banned_scripts:
+#                scripts: [ cyrillic ]
+#                max_percentage: 40
+
+            # Reject fields that contain more than 3 URLs, or repeat a single URL more than once
+#            url_count:
+#                max: 3
+#                max_identical: 1
```
Comment (kbond) [inline omines/antispam-bundle/0.1/config/packages/antispam.yaml:38]:
```suggestion
#                max_identical: 1

when@test:
    antispam:
        # disable in tests
        passive: true
```

I know passive mode via config is not yet implemented but what about adding this for when it is?

Comment (curry684) [reply omines/antispam-bundle/0.1/config/packages/antispam.yaml:38]: That is actually a brilliant idea. Thnx!

**Review by kbond — APPROVED** [review]
Looks good!

**Pull request merged**